backend/core/test/listings/listings.e2e-spec.ts

```diff
@@ -70,6 +70,11 @@ describe("Listings", () => {
     expect(res.body.items.map((listing) => listing.id).length).toBeGreaterThan(0)
   })
 
+  it("should return all external + internal listings", async () => {
```

Comment:
Have you run this test? I'm not sure that it will pass as-written

Reply:
Ooopss! Fixed!